engine/execution/state/state.go

```diff
@@ -334,7 +334,7 @@ func (s *state) PersistExecutionState(ctx context.Context, header *flow.Header,
 	spew.Config.DisableMethods = true
 	spew.Config.DisablePointerMethods = true
 
-	span, childCtx := s.tracer.StartSpanFromContext(ctx, trace.EXESaveExecutionResults)
+	span, childCtx := s.tracer.StartSpanFromContext(ctx, trace.EXEStatePersistExecutionState)
```

Comment:
I think old name was "more correct", we don't persist executions state here, we just save results (exec state is persisted async in commiter)

Reply:
I was trying to follow the name of the method , then we need to update that one as well.

Reply:
Yes, the method used to save execution state, both should be renamed then

Reply:
got it, going renaming both.